index.js

```diff
@@ -1,136 +1,38 @@
 /** @module google-maps-image-api */
+'use strict'
 
-var querystring = require( 'querystring' );
+const querystring = require( 'querystring' );
```

Comment:
Since this module is not being transpiled using `const` may break in older browsers.